forwards_paper.tex

```diff
@@ -1168,8 +1264,8 @@ \section*{Discussion}
 in \msprime{}, such as the coalescent with gene conversion \citep{Wiuf2000-rc}, using the structured coalescent to
 model various forms of natural selection \citep{Kaplan1988-in,Kaplan1989-rt,Braverman1995-gn},
 or the coalescent within a known pedigree.  For such models, one
-could in principle generate tables of nodes and edges to be simplified in \msprime{}.  The resulting succinct tree
-sequence object would be in the same format as those generated by \msprime{}'s built-in simulate function, and therefore
+could in principle generate tables of nodes and edges to be simplified in \tskit{}.  The resulting succinct tree
+sequence object would be in the same format as those generated by \tskit{}'s built-in simulate function, and therefore
```

Comment:
tskit -> msprime

Reply:
The 'built-in' is confusing here I think as we're switching back and forth between msprime and tskit. How about "The resulting succinct tree sequence object would be in the same format as those generated by \msprime{}'s \texttt{simulate} function, and ..."